db/schema.rb

```diff
@@ -35,12 +35,15 @@
   add_index "company_profiles", ["name"], name: "index_company_profiles_on_name", using: :btree
 
   create_table "follows", force: :cascade do |t|
-    t.integer "following_id", null: false
-    t.integer "follower_id",  null: false
+    t.integer  "followed_id",   null: false
+    t.string   "followed_type", null: false
```

Comment:
`_type` is the key to all polymorphic associations because it lets you associate to different models. If you have just `_id` then the id could belong to a `User` or `Project` model, and you would have no idea which one.